Signum.React/Scripts/SearchControl/SearchPage.tsx

```diff
@@ -78,7 +77,7 @@ function SearchPage(p: SearchPageProps) {
         allowSelection={qs && qs.allowSelection}
         hideFullScreenButton={true}
         largeToolbarButtons={true}
-        showFilters={SearchPage.showFilters(fo, qs)}
+        showFilters={qs?.showFilters ?? SearchPage.showFilters(fo, qs)}

```

Comment:
This is problematic because once set (to true) at the QuerySettings level it prevents UserQueries with PinnedFilters to work. 

I think if a UserQuery / FindOptions has pinned filters showFilters should be true, otherwise is not defined, sometimes you want to see them sometimes not. Right now this ambiguity can be  defined by overriding the extension point `SearchPage.showFilters` but will be good to have a general definitive solution.